demo/addons/fmod/fmod_config_window.tscn

```diff
@@ -0,0 +1,546 @@
+unique_name_in_owner = true
+margin_left = 162.0
+margin_right = 333.0
+margin_bottom = 24.0
+size_flags_horizontal = 3
+max_value = 16.0
+value = 2.0
+
+[node name="DSPBuffer" type="HBoxContainer" parent="Settings/Initialisation"]
+margin_top = 150.0
+margin_right = 333.0
+margin_bottom = 174.0
+
+[node name="Label" type="Label" parent="Settings/Initialisation/DSPBuffer"]
+margin_top = 5.0
+margin_right = 150.0
+margin_bottom = 19.0
+rect_min_size = Vector2( 150, 0 )
+hint_tooltip = "Size of DSP buffer (In samples)
+
+Smaller buffer size reduces latency, but increases CPU load and chance of playback errors. "
+mouse_filter = 1
+text = "DSP Buffer Size"
+align = 2
+
+[node name="VSeparator" type="VSeparator" parent="Settings/Initialisation/DSPBuffer"]
+margin_left = 154.0
+margin_right = 158.0
+margin_bottom = 24.0
+
+[node name="DSPBufferSize" type="SpinBox" parent="Settings/Initialisation/DSPBuffer"]
+unique_name_in_owner = true
+margin_left = 162.0
+margin_right = 333.0
+margin_bottom = 24.0
+size_flags_horizontal = 3
+min_value = 16.0
+max_value = 2048.0
+value = 1048.0
+
+[node name="ShowDebug" type="HBoxContainer" parent="Settings/Initialisation"]
+margin_top = 178.0
+margin_right = 333.0
+margin_bottom = 202.0
+
+[node name="Label" type="Label" parent="Settings/Initialisation/ShowDebug"]
+margin_top = 5.0
+margin_right = 150.0
+margin_bottom = 19.0
+rect_min_size = Vector2( 150, 0 )
+hint_tooltip = "Show FMOD debug overlay when running in game.
+
+WARNING: TURN OFF WHEN BUIDLING GAMES"
+mouse_filter = 1
+text = "Show Debug Panel"
+align = 2
+
+[node name="VSeparator" type="VSeparator" parent="Settings/Initialisation/ShowDebug"]
+margin_left = 154.0
+margin_right = 158.0
+margin_bottom = 24.0
+
+[node name="DebugChecker" type="CheckBox" parent="Settings/Initialisation/ShowDebug"]
+unique_name_in_owner = true
+margin_left = 162.0
+margin_right = 333.0
+margin_bottom = 24.0
+size_flags_horizontal = 3
+
+[node name="VSeparator" type="VSeparator" parent="Settings"]
+margin_left = 337.0
+margin_right = 341.0
+margin_bottom = 202.0
+
+[node name="Banks" type="VBoxContainer" parent="Settings"]
+margin_left = 345.0
+margin_right = 678.0
+margin_bottom = 202.0
+size_flags_horizontal = 3
+
+[node name="Banks" type="Label" parent="Settings/Banks"]
+margin_right = 333.0
+margin_bottom = 14.0
+text = "Banks"
+align = 1
+
+[node name="BankLocation" type="HBoxContainer" parent="Settings/Banks"]
+margin_top = 18.0
+margin_right = 333.0
+margin_bottom = 42.0
+alignment = 1
+
+[node name="Label" type="Label" parent="Settings/Banks/BankLocation"]
+margin_top = 5.0
+margin_right = 120.0
+margin_bottom = 19.0
+rect_min_size = Vector2( 120, 0 )
+text = "Bank Location"
+align = 2
+
+[node name="VSeparator" type="VSeparator" parent="Settings/Banks/BankLocation"]
+margin_left = 124.0
+margin_right = 128.0
+margin_bottom = 24.0
+
+[node name="BankFilePath" type="LineEdit" parent="Settings/Banks/BankLocation"]
```

Comment:
Don't use a LineEdit alone here, it's error-prone. Allow the user to select the folder using a file dialog, the same way it's made in the rest of Godot:
<img width="699" alt="image" src="https://user-images.githubusercontent.com/31470327/204178643-d86f5415-5bdb-4492-8160-d6b626562e92.png">